snakemake/io.py

```diff
@@ -1305,10 +1306,20 @@ def flatten(wildcard_values):
                     values = [values]
                 yield [(wildcard, value) for value in values]
 
+        def copy_flags(from_path, dest_path):
+            if hasattr(from_path, "flags"):
+                dest_path = AnnotatedString(dest_path)
+                dest_path.flags.update(from_path.flags)
+                if is_flagged(dest_path, "multiext"):
+                    dest_path.flags["multiext"] = self.apply_default_storage(
+                        self.replace_prefix(dest_path.flags["multiext"], property)
+                    )
+            return dest_path
```

Comment:
I fear that we forget the reason for this later. Can you add some comments?

Reply:
And why the special case for multiext?

Reply:
And maybe multiext should then also occur in the test case?

Reply:
Sure, I can add some comments.
Actually I am not sure if we need that special case for `multiext`.. I'll add it to the test case and check.